CMakeLists.txt

```diff
@@ -61,9 +61,13 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_FIND_PACKAGE_RESOLVE_SYMLINKS ON)
 set(CMAKE_INSTALL_MESSAGE LAZY)
-set(CMAKE_SKIP_RPATH ON)
 set(CMAKE_THREAD_PREFER_PTHREAD ON)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
+# this is use for building the python wheel
```

Comment:
The idea is to package the 
"libpartio.so.1", "partio.py",  "_partio.so", into a single wheel and therefore are unpakcing the _partio.so should look in the same directory for shared objects.

Reply:
I added an option to the cmake and the setup.py